ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/Runner.java

```diff
@@ -60,53 +61,55 @@
   private final HttpResponseFactory responseFactory = new HttpResponseFactory();
   private final Path dataDirectory;
 
-  private final Vertx vertx;
+  private Vertx vertx;
   private String jsonRpcHttpServiceId;
   private HttpServerService httpServerService;
 
   public Runner(
       final TransactionSerialiser serialiser,
-      final Vertx vertx,
       final HttpClientOptions clientOptions,
       final HttpServerOptions serverOptions,
       final Duration httpRequestTimeout,
       final TransactionFactory transactionFactory,
       final Path dataDirectory) {
     this.serialiser = serialiser;
     this.clientOptions = clientOptions;
+    LOG.debug("CLIENT OPTIONS: clientOptions.toString(): " + clientOptions.toString());
```

Comment:
Probably avoid doing ANYTHING in a constructor - can this go into the start()?

Reply:
Done